### PR TITLE
initial version of sym test

### PR DESCRIPTION
**Pull request by thecodeassassin**
Adds the `sym test` command

### CODE REVIEW

File: pkg/testing/test_runner.go [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+					},
+				},
+				RestartPolicy: v1.RestartPolicyNever,
+			},
+		}
+
+		num := i
+		executionStart := time.Now()
+
+		errGroup.Go(func() error {
```
Comment (Styren) [inline pkg/testing/test_runner.go:106]: This is a veeery long function, maybe it can be broken up a little bit?

**Review by Styren — APPROVED** [review]
LGTM just 1 small thing

File: pkg/testing/test_runner.go (hunk 1 above)
Comment (thecodeassassin) [reply pkg/testing/test_runner.go:121]: @Styren I split the method into two, can you check ?

Comment (Styren) [reply pkg/testing/test_runner.go:121]: Much better! 🙌

**Pull request merged**